src/deep-security/lib/Security.js

```diff
@@ -6,9 +6,11 @@
 
 import Kernel from 'deep-kernel';
 import {MissingLoginProviderException} from './Exception/MissingLoginProviderException';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['MissingLoginProviderException' is defined but never used.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1363209942/issues/source?bid=2895475&fileBranchId=2973909#l8)